client/extensions/woocommerce/lib/countries/US.js

```diff
@@ -0,0 +1,219 @@
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+export default () => {
+	return (
+		{
+			code: 'CA',
+			name: translate( 'Canada' ),
+			states: [
+				{
+					code: 'AL',
+					label: translate( 'Alabama' ),
+				},
+				{
+					code: 'AK',
+					label: translate( 'Alaska' ),
+				},
+				{
+					code: 'AZ',
+					label: translate( 'Arizona' ),
+				},
+				{
+					code: 'AR',
+					label: translate( 'Arkansas' ),
+				},
+				{
+					code: 'CA',
+					label: translate( 'California' ),
+				},
+				{
+					code: 'CO',
+					label: translate( 'Colorado' ),
+				},
+				{
+					code: 'CT',
+					label: translate( 'Connecticut' ),
+				},
+				{
+					code: 'DE',
+					label: translate( 'Delaware' ),
+				},
+				{
+					code: 'DC',
+					label: translate( 'District Of Columbia' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'District of Columbia' )` *ES Score: 22*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).